docs/tutorials/controlling-wedo2-motor.md

```diff
@@ -65,10 +65,15 @@ internal bluetooth and the new USB one:
 	    LMP Version: 4.0 (0x6)  Subversion: 0x220e
 	    Manufacturer: Broadcom Corporation (15)
 
-In the above situation, `hci0` is our Bluetooth 4.0 BLE device (note "BUS: USB" and
-"HCI version: 4.0"). If you don't see status "UP RUNNING" you need to activate
-Bluetooth first (one easy way is using the Brickman User Interface: choose "Wireless
-and Networks" at the main screen then "Bluetooth")
+In the above situation, `hci0` is our Bluetooth 4.0 BLE device (note "BUS: USB").
+Initial releases of EV3 bricks will show older HCI/LMP versions for the internal
+USB device (`hci1` here) but it always be shown with `BUS: UART`. Although the
+newer releases show a HCI/LMP 4.0 version this new chipset doesn't include the
+Bluetooth Low Energy subset ot the BT 4.0 standard.
+
+If you don't see status "UP RUNNING" you will need to activate Bluetooth 
+first (one easy way is using the Brickman User Interface: choose "Wireless and 
+Networks" at the main screen then "Bluetooth").
```

Comment:
Might be worth mentioning that you can run `connmanctl enable bluetooth` from a command line as well.